frontend/src/app/_components/creator-list/creator-list.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { Creator } from '@app/_models';
+import { Observable } from 'rxjs';
+
+@Component({
+  selector: 'app-creator-list',
+  templateUrl: './creator-list.component.html',
+  styleUrls: ['./creator-list.component.scss']
+})
+export class CreatorListComponent implements OnInit {
+  @Input() creators!: Creator[];
```

Comment:
Just set it to [] instead of declaring it ! Maybe? That way it won't crash if they don't pass in a list, it'll just be empty.